langchain4j-vertex-ai/src/main/java/dev/langchain4j/model/vertexai/VertexAiImageModel.java

```diff
@@ -270,7 +435,12 @@ public Response<Image> edit(Image image, Image mask, String prompt) {
     private URI persistAndGetURI(String bytesBase64Encoded) {
         if (this.withPersisting != null && this.withPersisting) {
             try {
-                Path tempFile = Files.createTempFile(this.tempDirectory, "imagen-image-", ".png");
+                String suffix = ".png";
+                if (this.mimeType != null && this.mimeType == MimeType.JPEG) {
```

Comment:
```suggestion
                if (this.mimeType == MimeType.JPEG) {
```
It is safe to compare null to enum